app/presenters/document/record_presenter.rb

```diff
@@ -33,6 +33,39 @@ def creator_title
       end
     end
 
+    ##
+    # Looks up dc:Creators, if they are europeana entity agents returns them paired with
+    # the respective urls for retrieving thumbnails via JS.
+    #
+    # @return [Hash]
+    def creators_info
+      creator_entities = agents_for('proxies.dcCreator').select do |entity|
+        URI.parse(entity['about']).host == 'data.europeana.eu'
```

Comment:
We will need to rescue exceptions from `URI.parse` as agent URIs may be in a format it does not understand.

Reply:
... or just use `entity['about'].include?('://data.europeana.eu/')